doc.ddoc

```diff
@@ -1,3 +1,4 @@
+ROOT_DIR =
```

Comment:
But you override it anyhow, so keeping the default in sync with ROOT seems to make sense.
I don't care too much about this though.